webgpu.h

```diff
@@ -797,6 +836,10 @@ typedef void (*WGPUProcRenderPassEncoderSetStencilReference)(WGPURenderPassEncod
 typedef void (*WGPUProcRenderPassEncoderSetVertexBuffers)(WGPURenderPassEncoder renderPassEncoder, uint32_t startSlot, uint32_t count, WGPUBuffer const * buffers, uint64_t const * offsets);
 typedef void (*WGPUProcRenderPassEncoderSetViewport)(WGPURenderPassEncoder renderPassEncoder, float x, float y, float width, float height, float minDepth, float maxDepth);
 
+// Procs of SwapChain
+typedef WGPUTextureView (*WGPUProcSwapChainGetCurrentTextureView)(WGPUSwapChain swapChain);
```

Comment:
From the F2F, I thought we were going to go with Texture instead of TextureView here?

Reply:
From the unofficial F2F, @Kangz noted they'd be fine with restricting the native API to be render-only, so returning `TextureView` makes sense. Let me know if you want to reconsider that.

Our implementation would for sure prefer the swapchain to be render-only.

Reply:
Gotcha, this is fine with me.

Reply:
Yeah the main argument for WebGPU keeping `getTexture` is because we need to be able to do readbacks on the Web anyway. But in native we want to take advantage of "renderOnly" textures that can be more optimized.